traffic_ops/traffic_ops_golang/deliveryservice/request/requests.go

```diff
@@ -716,6 +740,18 @@ func putV40(w http.ResponseWriter, r *http.Request, inf *api.APIInfo) (result ds
 		dsr.Status,
 		inf.IntParams["id"],
 	}
+	if dsr.Original != nil {
+		if dsr.Original.LongDesc1 != nil || dsr.Original.LongDesc2 != nil {
+			api.HandleErr(w, r, tx, http.StatusBadRequest, errors.New("the Long Description 1 and Long Description 2 fields are no longer supported in API 4.0 onwards"), nil)
```

Comment:
this error message should probably use the json field names, i.e. `longDesc1` and `longDesc2` just to be clear